bot/pagination.py

```diff
@@ -10,7 +10,7 @@
 LEFT_EMOJI = "\u2B05"    # [:arrow_left:]
 RIGHT_EMOJI = "\u27A1"   # [:arrow_right:]
 LAST_EMOJI = "\u23ED"    # [:track_next:]
-DELETE_EMOJI = "\u274c"  # [:x:]
+DELETE_EMOJI = "<:trashcan:637136429717389331>"  # [:trashcan:]
 
 PAGINATION_EMOJI = [FIRST_EMOJI, LEFT_EMOJI, RIGHT_EMOJI, LAST_EMOJI, DELETE_EMOJI]
```

Comment:
A minor changes & considerations. Otherwise the functionality works well.
This can be changed from a list to a tuple instead.

Reply:
Sure, will do. I just saw that this is the implementation at https://github.com/python-discord/bot
Should I also apply this migration in the /python-discord/bot?

Reply:
Should, yes, but doing a whole PR just to change a line, I don't think it's worth it, unless you want to refactor it, then yes, please do!

Reply:
Done, waiting for your approval on both bot repos 👍